backend/gamify/verification/views.py

```diff
@@ -1,38 +1,114 @@
+import os
+from datetime import datetime, time
+
 from django.http import JsonResponse
+from django.core.files.storage import FileSystemStorage
 from rest_framework.decorators import api_view
 
+from gamify.models import User
+
+from azure.cognitiveservices.vision.face import FaceClient
+from msrest.authentication import CognitiveServicesCredentials
+from google.cloud import vision
+
+AZURE_KEY = os.environ['AZURE_KEY']
+AZURE_ENDPOINT = 'https://lahacks.cognitiveservices.azure.com/'
```

Comment:
Will you remove Azure in this PR or the next?

Reply:
Probably in the next PR along with facial recognition